numalogic/transforms/__init__.py

```diff
@@ -9,6 +9,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from numalogic.preprocess.transformer import LogTransformer, StaticPowerTransformer, TanhScaler
+"""
+Module to provide timeseries transformations needed for preprocessing,
+feature engineering and postprocessing.
+"""
 
-__all__ = ["LogTransformer", "StaticPowerTransformer", "TanhScaler"]
+from numalogic.transforms._scaler import TanhScaler
```

Comment:
should we name the folder transformers?

Reply:
Thought about it, and here are my thoughts.

1. Transformer on ML terms defaults to the transformer network architecture; so it is important to avoid that confusion there
2. Alternatives could be `transformations`, `datatransform` or `transform`. But shorter is always better.